assets/js/blocks/handpicked-products/block.json

```diff
@@ -28,6 +28,13 @@
         },
         "contentVisibility": {
             "type": "object",
+            "default": {
```

Comment:
Just double-checking, are changes to the `Handpicked Products` block intended in this `Products by Tag` PR?

Reply:
@frontdevde I just found that was missing while doing this PR and included here the change 😅